harness/determined/ipc.py

```diff
@@ -133,36 +141,49 @@ def broadcast(self, obj: Any) -> None:
         self._pub_socket.send_pyobj(_SerialMessage(self._send_serial, obj))
         self._send_serial += 1
 
-    def gather_with_polling(self, health_check: Callable[[], None]) -> List[Any]:
+    def gather_with_polling(self, health_check: Callable[[], None]) -> Tuple[List[Any], bool]:
         """
         Gather a response message from each connection, with a health_check callback that can raise
         an error if something goes wrong.
         """
         return self._recv_all_with_polling(health_check)
 
-    def _recv_all_with_polling(self, health_check: Callable[[], None]) -> List[Any]:
+    def _recv_all_with_polling(self, health_check: Callable[[], None]) -> Tuple[List[Any], bool]:
```

Comment:
If you push again, probably just remove this function, since it is basically an alias for `gather_with_polling()`.  Not sure how that situation arose... but whatever.